### PR TITLE
add project count in analysisframework schema

### PR DESCRIPTION
**Pull request by sudan45**
- Addresses #1403 
## Changes
- Add project count in analysisframework schema 
- include test project
## This PR doesn't introduce any:

- [X] temporary files, auto-generated files or secret keys
- [X] n+1 queries
- [X] flake8 issues
- [X] `print`
- [X] typos
- [X] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: apps/analysis_framework/schema.py [hunk 1]
```diff
@@ -122,6 +122,7 @@ class Meta:
         ),
         required=True,
     )
+    used_in_project_count = graphene.Int(required=False)
```
Comment (thenav56) [inline apps/analysis_framework/schema.py]:
```suggestion
    used_in_project_count = graphene.Int(required=True)
```

File: apps/analysis_framework/schema.py [hunk 2]
```diff
@@ -147,6 +148,10 @@ def resolve_allowed_permissions(root, info):
     def resolve_tags(root, info):
         return info.context.dl.analysis_framework.af_tags.load(root.id)
 
+    @staticmethod
+    def resolve_used_in_project_count(root, info):
+        return Project.objects.filter(analysis_framework=root.id, is_test=False).count()
```
Comment (thenav56) [inline apps/analysis_framework/schema.py]: Let's define Dataloader.

Do we need is_test filter here? @AdityaKhatri @subinasr

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: apps/analysis_framework/schema.py (hunk 2 above)
Comment (subinasr) [reply apps/analysis_framework/schema.py]: It's not necessary.

Comment (subinasr) [reply apps/analysis_framework/schema.py]: Reiteration: Yes, we do need this filter @sudan45 @thenav56

File: apps/analysis_framework/schema.py [hunk 3]
```diff
@@ -122,6 +127,7 @@ class Meta:
         ),
         required=True,
     )
+    used_in_project_count = graphene.Field(AnalysisFrameworkProjectCount)
```
Comment (thenav56) [inline apps/analysis_framework/schema.py]:
```suggestion
    used_in_project_count = graphene.Field(AnalysisFrameworkProjectCount, required=True)
```

File: apps/analysis_framework/schema.py [hunk 4]
```diff
@@ -147,6 +153,11 @@ def resolve_allowed_permissions(root, info):
     def resolve_tags(root, info):
         return info.context.dl.analysis_framework.af_tags.load(root.id)
 
+    @staticmethod
+    def resolve_used_in_project_count(root, info):
+        return info.context.dl.analysis_framework.af_project_count.load(root.id)
+        # return Project.objects.filter(analysis_framework=root.id, is_test=False).count()
```
Comment (thenav56) [inline apps/analysis_framework/schema.py]:
```suggestion
```

File: schema.graphql
Comment (thenav56) [inline schema.graphql:1]: Let's re-run schema generate

**Review by thenav56 — CHANGES_REQUESTED** [review]

**Review by thenav56 — APPROVED** [review]

**Review by subinasr — APPROVED** [review]

**Pull request merged**